components/tabs/src/sync/engine.rs

```diff
@@ -161,7 +175,7 @@ impl<'a> SyncEngine for TabsEngine<'a> {
             let local_record = ClientRemoteTabs {
                 client_id: local_id,
                 client_name,
-                device_type,
+                device_type: device_type.into(),
```

Comment:
using the clone of `DeviceType` in storage.rs will avoid this into (and I think is just generally cleaner - there doesn't seem to be a good reason for this crate to have both versions). That might even mean you can avoid that `From` impl too.

Reply:
Per our conversation, this won't be possible since we get `remote_client.device_type` from Sync15 and cannot directly use the local `DeviceType`